velox/docs/functions/spark/misc_function.rst

```diff
@@ -0,0 +1,9 @@
+====================================
+Miscellaneous Functions
+====================================
+.. spark:function:: spark_partition_id() -> integer
```

Comment:
add empty line between the title and the first function

Reply:
Upated, thanks.